### ISSUE TITLE
ICE DRIFT (DIRECTION) Graphic Not Generating KML

### ISSUE THREAD

**Issue by david-melendez** [missioncommand/old-mil-sym-java#34]
I am attempting to generate KML for the ICE DRIFT (DIRECTION) graphic, but the renderer is returning empty KML.  Here is the call I am making:

```
String kml = r.RenderMultiPointSymbol("ID", "ICE DRIFT",
                "ICE DRIFT", "WO-DIDID---L---", 
                "-86.5870132446289,30.6727600097656 -86.5819625854492,30.6749858856201 -86.5769119262695,30.677209854126 -86.5658798217773,30.6610260009766 -86.5837783813477,30.6588668823242 -86.5914916992188,30.6669597625732 ",
                "clampToGround", 50000, "", 
                "",  1);
```

Here is the KML I am getting back.

`<Folder id="ID"><name><![CDATA[ICE DRIFT]]></name><visibility>1</visibility></Folder>`

If I change the ID, the call works just fine.  For example, the code below works.

```
String kml = r.RenderMultiPointSymbol("ID", "ICE DRIFT",
                "ICE DRIFT", "WO-DILOV---L---", 
                "-86.5870132446289,30.6727600097656 -86.5819625854492,30.6749858856201 -86.5769119262695,30.677209854126 -86.5658798217773,30.6610260009766 -86.5837783813477,30.6588668823242 -86.5914916992188,30.6669597625732 ",
                "clampToGround", 50000, "", 
                "", 0, 1);
```

Here is the KML generated:

```
<Folder id="ID">
	<name>
		<![CDATA[ICE DRIFT]]>
	</name>
	<visibility>1</visibility>
	<Placemark>
		<description>
			<![CDATA[<b>ICE DRIFT</b><br/>
ICE DRIFT]]>
		</description>
		<Style id="lineColor">
			<LineStyle>
				<color>ff000000</color>
				<colorMode>normal</colorMode>
				<width>3</width>
			</LineStyle>
		</Style>
		<MultiGeometry>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.58706261,30.67233364 -86.58742829,30.67216626 -86.58761499,30.6722154 -86.5878665,30.67236947 -86.5878665,30.67236947 -86.58780941,30.67253019 -86.58781713,30.67279584 -86.58745145,30.67296321 -86.58727247,30.67317973 -86.58696387,30.67318638 -86.58659819,30.67335376 -86.58641149,30.67330462 -86.58615999,30.67315055 -86.58615999,30.67315055 -86.58615227,30.6728849 -86.58620936,30.67272418 -86.58657504,30.67255681 -86.58675402,30.67234029 -86.58706261,30.67233364 -86.58742829,30.67216626 -86.5876798,30.67232033 -86.5878665,30.67236947 -86.5878665,30.67236947 -86.58780941,30.67253019 -86.58781713,30.67279584 -86.58745145,30.67296321 -86.58727247,30.67317973 -86.58696387,30.67318638 -86.58659819,30.67335376 -86.58634669,30.67319969 -86.58615999,30.67315055 -86.58615999,30.67315055 -86.58621708,30.67298983 -86.58620936,30.67272418 -86.58657504,30.67255681 -86.58675402,30.67234029 -86.58675402,30.67234029</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.58201198,30.67455951 -86.58237767,30.67439214 -86.58256437,30.67444128 -86.58281588,30.67459534 -86.58281588,30.67459534 -86.58275878,30.67475606 -86.58276649,30.67502171 -86.58240079,30.67518909 -86.58222179,30.6754056 -86.58191319,30.67541226 -86.58154749,30.67557963 -86.58136079,30.67553049 -86.58110929,30.67537643 -86.58110929,30.67537643 -86.58110159,30.67511078 -86.58115868,30.67495006 -86.58152438,30.67478268 -86.58170338,30.67456617 -86.58201198,30.67455951 -86.58237767,30.67439214 -86.58262918,30.67454621 -86.58281588,30.67459534 -86.58281588,30.67459534 -86.58275878,30.67475606 -86.58276649,30.67502171 -86.58240079,30.67518909 -86.58222179,30.6754056 -86.58191319,30.67541226 -86.58154749,30.67557963 -86.58129599,30.67542557 -86.58110929,30.67537643 -86.58110929,30.67537643 -86.58116639,30.67521571 -86.58115868,30.67495006 -86.58152438,30.67478268 -86.58170338,30.67456617 -86.58170338,30.67456617</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.57740914,30.67722996 -86.57762241,30.67753556 -86.57757516,30.67769863 -86.57740955,30.67792291 -86.57740955,30.67792291 -86.57722012,30.67788225 -86.57691233,30.6779028 -86.57669906,30.67759721 -86.57643854,30.67745468 -86.57641471,30.67718974 -86.57620144,30.67688415 -86.5762487,30.67672108 -86.57641431,30.6764968 -86.57641431,30.6764968 -86.57672209,30.67647625 -86.57691152,30.67651691 -86.57712479,30.6768225 -86.57738531,30.67696503 -86.57740914,30.67722996 -86.57762241,30.67753556 -86.57745681,30.67775984 -86.57740955,30.67792291 -86.57740955,30.67792291 -86.57722012,30.67788225 -86.57691233,30.6779028 -86.57669906,30.67759721 -86.57643854,30.67745468 -86.57641471,30.67718974 -86.57620144,30.67688415 -86.57636705,30.67665987 -86.57641431,30.6764968 -86.57641431,30.6764968 -86.57660374,30.67653745 -86.57691152,30.67651691 -86.57712479,30.6768225 -86.57738531,30.67696503 -86.57738531,30.67696503</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.57557019,30.67453266 -86.57578345,30.67483825 -86.57573619,30.67500132 -86.57557059,30.6752256 -86.57557059,30.6752256 -86.57538115,30.67518494 -86.57507338,30.67520549 -86.57486012,30.6748999 -86.57459961,30.67475737 -86.57457578,30.67449244 -86.57436253,30.67418684 -86.57440979,30.67402377 -86.57457539,30.67379949 -86.57457539,30.67379949 -86.57488317,30.67377894 -86.57507259,30.6738196 -86.57528585,30.67412519 -86.57554636,30.67426772 -86.57557019,30.67453266 -86.57578345,30.67483825 -86.57561785,30.67506253 -86.57557059,30.6752256 -86.57557059,30.6752256 -86.57538115,30.67518494 -86.57507338,30.67520549 -86.57486012,30.6748999 -86.57459961,30.67475737 -86.57457578,30.67449244 -86.57436253,30.67418684 -86.57452814,30.67396256 -86.57457539,30.67379949 -86.57457539,30.67379949 -86.57476482,30.67384015 -86.57507259,30.6738196 -86.57528585,30.67412519 -86.57554636,30.67426772 -86.57554636,30.67426772</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.57373134,30.67183535 -86.57394458,30.67214094 -86.57389733,30.67230402 -86.57373172,30.67252829 -86.57373172,30.67252829 -86.5735423,30.67248764 -86.57323453,30.67250818 -86.57302129,30.67220259 -86.57276078,30.67206006 -86.57273696,30.67179513 -86.57252372,30.67148953 -86.57257098,30.67132646 -86.57273659,30.67110218 -86.57273659,30.67110218 -86.57304435,30.67108163 -86.57323377,30.67112229 -86.57344701,30.67142788 -86.57370752,30.67157041 -86.57373134,30.67183535 -86.57394458,30.67214094 -86.57377898,30.67236522 -86.57373172,30.67252829 -86.57373172,30.67252829 -86.5735423,30.67248764 -86.57323453,30.67250818 -86.57302129,30.67220259 -86.57276078,30.67206006 -86.57273696,30.67179513 -86.57252372,30.67148953 -86.57268932,30.67126525 -86.57273659,30.67110218 -86.57273659,30.67110218 -86.57292601,30.67114284 -86.57323377,30.67112229 -86.57344701,30.67142788 -86.57370752,30.67157041 -86.57370752,30.67157041</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.57189259,30.66913804 -86.57210582,30.66944363 -86.57205856,30.66960671 -86.57189296,30.66983099 -86.57189296,30.66983099 -86.57170354,30.66979033 -86.57139578,30.66981088 -86.57118255,30.66950528 -86.57092205,30.66936275 -86.57089824,30.66909782 -86.57068501,30.66879222 -86.57073227,30.66862915 -86.57089788,30.66840487 -86.57089788,30.66840487 -86.57120563,30.66838432 -86.57139505,30.66842498 -86.57160828,30.66873058 -86.57186877,30.6688731 -86.57189259,30.66913804 -86.57210582,30.66944363 -86.57194022,30.66966791 -86.57189296,30.66983099 -86.57189296,30.66983099 -86.57170354,30.66979033 -86.57139578,30.66981088 -86.57118255,30.66950528 -86.57092205,30.66936275 -86.57089824,30.66909782 -86.57068501,30.66879222 -86.57085061,30.66856794 -86.57089788,30.66840487 -86.57089788,30.66840487 -86.57108729,30.66844553 -86.57139505,30.66842498 -86.57160828,30.66873058 -86.57186877,30.6688731 -86.57186877,30.6688731</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.57005394,30.66644073 -86.57026716,30.66674632 -86.5702199,30.6669094 -86.5700543,30.66713368 -86.5700543,30.66713368 -86.56986489,30.66709302 -86.56955713,30.66711357 -86.56934391,30.66680797 -86.56908343,30.66666545 -86.56905962,30.66640051 -86.5688464,30.66609491 -86.56889367,30.66593184 -86.56905927,30.66570756 -86.56905927,30.66570756 -86.56936702,30.66568701 -86.56955643,30.66572767 -86.56976965,30.66603327 -86.57003013,30.66617579 -86.57005394,30.66644073 -86.57026716,30.66674632 -86.57010156,30.6669706 -86.5700543,30.66713368 -86.5700543,30.66713368 -86.56986489,30.66709302 -86.56955713,30.66711357 -86.56934391,30.66680797 -86.56908343,30.66666545 -86.56905962,30.66640051 -86.5688464,30.66609491 -86.56901201,30.66587063 -86.56905927,30.66570756 -86.56905927,30.66570756 -86.56924868,30.66574822 -86.56955643,30.66572767 -86.56976965,30.66603327 -86.57003013,30.66617579 -86.57003013,30.66617579</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.5682154,30.66374342 -86.56842861,30.66404902 -86.56838134,30.66421209 -86.56821574,30.66443637 -86.56821574,30.66443637 -86.56802633,30.66439571 -86.56771859,30.66441626 -86.56750538,30.66411066 -86.56724491,30.66396814 -86.5672211,30.6637032 -86.5670079,30.6633976 -86.56705517,30.66323453 -86.56722077,30.66301025 -86.56722077,30.66301025 -86.56752851,30.6629897 -86.56771791,30.66303036 -86.56793112,30.66333596 -86.56819159,30.66347848 -86.5682154,30.66374342 -86.56842861,30.66404902 -86.56826301,30.6642733 -86.56821574,30.66443637 -86.56821574,30.66443637 -86.56802633,30.66439571 -86.56771859,30.66441626 -86.56750538,30.66411066 -86.56724491,30.66396814 -86.5672211,30.6637032 -86.5670079,30.6633976 -86.5671735,30.66317332 -86.56722077,30.66301025 -86.56722077,30.66301025 -86.56741017,30.66305091 -86.56771791,30.66303036 -86.56793112,30.66333596 -86.56819159,30.66347848 -86.56819159,30.66347848</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.56609567,30.66063987 -86.56650575,30.66059025 -86.56666163,30.6606914 -86.56683668,30.66091023 -86.56683668,30.66091023 -86.56671917,30.66104445 -86.56662084,30.66129635 -86.56621075,30.66134597 -86.56595654,30.66149673 -86.56566397,30.66141213 -86.56525388,30.66146175 -86.56509801,30.6613606 -86.56492296,30.66114178 -86.56492296,30.66114178 -86.5650213,30.66088987 -86.56513881,30.66075565 -86.5655489,30.66070603 -86.56580311,30.66055527 -86.56609567,30.66063987 -86.56650575,30.66059025 -86.56668081,30.66080908 -86.56683668,30.66091023 -86.56683668,30.66091023 -86.56671917,30.66104445 -86.56662084,30.66129635 -86.56621075,30.66134597 -86.56595654,30.66149673 -86.56566397,30.66141213 -86.56525388,30.66146175 -86.56507883,30.66124292 -86.56492296,30.66114178 -86.56492296,30.66114178 -86.56504048,30.66100755 -86.56513881,30.66075565 -86.5655489,30.66070603 -86.56580311,30.66055527 -86.56580311,30.66055527</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.56967543,30.66020805 -86.57008551,30.66015843 -86.57024139,30.66025957 -86.57041645,30.6604784 -86.57041645,30.6604784 -86.57029894,30.66061262 -86.57020062,30.66086453 -86.56979054,30.66091415 -86.56953633,30.66106491 -86.56924376,30.6609803 -86.56883368,30.66102992 -86.5686778,30.66092878 -86.56850274,30.66070995 -86.56850274,30.66070995 -86.56860107,30.66045805 -86.56871858,30.66032383 -86.56912866,30.66027421 -86.56938286,30.66012344 -86.56967543,30.66020805 -86.57008551,30.66015843 -86.57026057,30.66037726 -86.57041645,30.6604784 -86.57041645,30.6604784 -86.57029894,30.66061262 -86.57020062,30.66086453 -86.56979054,30.66091415 -86.56953633,30.66106491 -86.56924376,30.6609803 -86.56883368,30.66102992 -86.56865862,30.6608111 -86.56850274,30.66070995 -86.56850274,30.66070995 -86.56862025,30.66057573 -86.56871858,30.66032383 -86.56912866,30.66027421 -86.56938286,30.66012344 -86.56938286,30.66012344</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.57325516,30.65977623 -86.57366523,30.65972661 -86.57382111,30.65982775 -86.57399618,30.66004658 -86.57399618,30.66004658 -86.57387868,30.6601808 -86.57378037,30.66043271 -86.57337029,30.66048232 -86.57311609,30.66063309 -86.57282352,30.66054848 -86.57241344,30.6605981 -86.57225756,30.66049695 -86.5720825,30.66027813 -86.5720825,30.66027813 -86.57218082,30.66002622 -86.57229832,30.659892 -86.5727084,30.65984238 -86.57296259,30.65969162 -86.57325516,30.65977623 -86.57366523,30.65972661 -86.5738403,30.65994544 -86.57399618,30.66004658 -86.57399618,30.66004658 -86.57387868,30.6601808 -86.57378037,30.66043271 -86.57337029,30.66048232 -86.57311609,30.66063309 -86.57282352,30.66054848 -86.57241344,30.6605981 -86.57223838,30.66037927 -86.5720825,30.66027813 -86.5720825,30.66027813 -86.5722,30.66014391 -86.57229832,30.659892 -86.5727084,30.65984238 -86.57296259,30.65969162 -86.57296259,30.65969162</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.57683486,30.6593444 -86.57724493,30.65929478 -86.57740081,30.65939593 -86.57757589,30.65961475 -86.57757589,30.65961475 -86.57745839,30.65974898 -86.57736009,30.66000088 -86.57695001,30.6600505 -86.57669582,30.66020126 -86.57640325,30.66011666 -86.57599317,30.66016628 -86.57583729,30.66006513 -86.57566222,30.65984631 -86.57566222,30.65984631 -86.57576053,30.6595944 -86.57587802,30.65946018 -86.5762881,30.65941056 -86.57654228,30.6592598 -86.57683486,30.6593444 -86.57724493,30.65929478 -86.57742,30.65951361 -86.57757589,30.65961475 -86.57757589,30.65961475 -86.57745839,30.65974898 -86.57736009,30.66000088 -86.57695001,30.6600505 -86.57669582,30.66020126 -86.57640325,30.66011666 -86.57599317,30.66016628 -86.5758181,30.65994745 -86.57566222,30.65984631 -86.57566222,30.65984631 -86.57577972,30.65971208 -86.57587802,30.65946018 -86.5762881,30.65941056 -86.57654228,30.6592598 -86.57654228,30.6592598</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.58041452,30.65891258 -86.58082459,30.65886296 -86.58098047,30.6589641 -86.58115556,30.65918293 -86.58115556,30.65918293 -86.58103807,30.65931715 -86.58093977,30.65956906 -86.5805297,30.65961868 -86.58027552,30.65976944 -86.57998294,30.65968483 -86.57957287,30.65973445 -86.57941699,30.65963331 -86.57924191,30.65941448 -86.57924191,30.65941448 -86.5793402,30.65916258 -86.5794577,30.65902835 -86.57986777,30.65897874 -86.58012195,30.65882797 -86.58041452,30.65891258 -86.58082459,30.65886296 -86.58099967,30.65908179 -86.58115556,30.65918293 -86.58115556,30.65918293 -86.58103807,30.65931715 -86.58093977,30.65956906 -86.5805297,30.65961868 -86.58027552,30.65976944 -86.57998294,30.65968483 -86.57957287,30.65973445 -86.57939779,30.65951562 -86.57924191,30.65941448 -86.57924191,30.65941448 -86.5793594,30.65928026 -86.5794577,30.65902835 -86.57986777,30.65897874 -86.58012195,30.65882797 -86.58012195,30.65882797</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.58427304,30.65881979 -86.5845392,30.6590929 -86.58452217,30.65926032 -86.5843994,30.65950414 -86.5843994,30.65950414 -86.58420494,30.65948949 -86.58390474,30.65955123 -86.58363858,30.65927812 -86.58335541,30.65917244 -86.58328372,30.65891398 -86.58301757,30.65864087 -86.5830346,30.65847344 -86.58315737,30.65822963 -86.58315737,30.65822963 -86.58345757,30.65816789 -86.58365203,30.65818253 -86.58391818,30.65845564 -86.58420135,30.65856133 -86.58427304,30.65881979 -86.5845392,30.6590929 -86.58441643,30.65933671 -86.5843994,30.65950414 -86.5843994,30.65950414 -86.58420494,30.65948949 -86.58390474,30.65955123 -86.58363858,30.65927812 -86.58335541,30.65917244 -86.58328372,30.65891398 -86.58301757,30.65864087 -86.58314034,30.65839705 -86.58315737,30.65822963 -86.58315737,30.65822963 -86.58335183,30.65824427 -86.58365203,30.65818253 -86.58391818,30.65845564 -86.58420135,30.65856133 -86.58420135,30.65856133</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.58684402,30.66151742 -86.58711019,30.66179053 -86.58709317,30.66195795 -86.5869704,30.66220176 -86.5869704,30.66220176 -86.58677593,30.66218712 -86.58647572,30.66224886 -86.58620955,30.66197575 -86.58592637,30.66187006 -86.58585467,30.6616116 -86.5855885,30.66133849 -86.58560553,30.66117107 -86.5857283,30.66092725 -86.5857283,30.66092725 -86.5860285,30.66086551 -86.58622297,30.66088016 -86.58648913,30.66115327 -86.58677232,30.66125895 -86.58684402,30.66151742 -86.58711019,30.66179053 -86.58698742,30.66203434 -86.5869704,30.66220176 -86.5869704,30.66220176 -86.58677593,30.66218712 -86.58647572,30.66224886 -86.58620955,30.66197575 -86.58592637,30.66187006 -86.58585467,30.6616116 -86.5855885,30.66133849 -86.58571127,30.66109468 -86.5857283,30.66092725 -86.5857283,30.66092725 -86.58592276,30.6609419 -86.58622297,30.66088016 -86.58648913,30.66115327 -86.58677232,30.66125895 -86.58677232,30.66125895</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.58941514,30.66421504 -86.58968132,30.66448815 -86.5896643,30.66465558 -86.58954154,30.66489939 -86.58954154,30.66489939 -86.58934707,30.66488474 -86.58904685,30.66494648 -86.58878067,30.66467337 -86.58849747,30.66456769 -86.58842576,30.66430923 -86.58815958,30.66403612 -86.5881766,30.66386869 -86.58829937,30.66362488 -86.58829937,30.66362488 -86.58859958,30.66356314 -86.58879405,30.66357779 -86.58906023,30.6638509 -86.58934343,30.66395658 -86.58941514,30.66421504 -86.58968132,30.66448815 -86.58955856,30.66473197 -86.58954154,30.66489939 -86.58954154,30.66489939 -86.58934707,30.66488474 -86.58904685,30.66494648 -86.58878067,30.66467337 -86.58849747,30.66456769 -86.58842576,30.66430923 -86.58815958,30.66403612 -86.58828235,30.66379231 -86.58829937,30.66362488 -86.58829937,30.66362488 -86.58849384,30.66363953 -86.58879405,30.66357779 -86.58906023,30.6638509 -86.58934343,30.66395658 -86.58934343,30.66395658</coordinates>
			</LineString>
			<LineString>
				<tessellate>1</tessellate>
				<altitudeMode>clampToGround</altitudeMode>
				<coordinates>-86.5919864,30.66691267 -86.5922526,30.66718578 -86.59223558,30.6673532 -86.59211282,30.66759702 -86.59211282,30.66759702 -86.59191835,30.66758237 -86.59161812,30.66764411 -86.59135192,30.667371 -86.59106871,30.66726532 -86.590997,30.66700686 -86.5907308,30.66673375 -86.59074782,30.66656632 -86.59087058,30.66632251 -86.59087058,30.66632251 -86.5911708,30.66626077 -86.59136528,30.66627541 -86.59163147,30.66654852 -86.59191468,30.66665421 -86.5919864,30.66691267 -86.5922526,30.66718578 -86.59212984,30.66742959 -86.59211282,30.66759702 -86.59211282,30.66759702 -86.59191835,30.66758237 -86.59161812,30.66764411 -86.59135192,30.667371 -86.59106871,30.66726532 -86.590997,30.66700686 -86.5907308,30.66673375 -86.59085357,30.66648993 -86.59087058,30.66632251 -86.59087058,30.66632251 -86.59106506,30.66633716 -86.59136528,30.66627541 -86.59163147,30.66654852 -86.59191468,30.66665421 -86.59191468,30.66665421</coordinates>
			</LineString>
		</MultiGeometry>
	</Placemark>
</Folder>
```

**Comment by mdeutch3** [comment]
Ice Drift, symbol Id WO-DIDID---L--- was missing from the symbol dictionary so it would not render. The symbol has been added to the dictionary and should render.

**Comment by michael-spinelli** [comment]
fixed in release v0.1.16
